src/codegen/properties/function_parameters.py

```diff
@@ -0,0 +1,93 @@
+"""Structure for storing function parameters metadata from scrapigen."""
+from codegen.properties.cluster_elements import ClusterElements
+
+
+class FunctionsParameter:
+    """Structure for storing function metadata from scrapigen."""
+
+    def __init__(self, parameters_metadata):
+        """Structure for storing function metadata from scrapigen."""
+        self._direction = parameters_metadata["direction"]
+        self._parameters_name = parameters_metadata["name"]
+        self._type = parameters_metadata["type"]
+        self._ctypes_data_type = parameters_metadata["ctypes_data_type"]
+        self._python_data_type = parameters_metadata["python_data_type"]
+        self._description = parameters_metadata["description"]
+        self._is_list = parameters_metadata.get("is_list", False)
+        self._has_explicit_buffer_size = parameters_metadata.get("has_explicit_buffer_size", False)
+        self._optional = parameters_metadata.get("optional", False)
+        self._default = parameters_metadata.get("default", "")
+        self._cluster = parameters_metadata.get("cluster", "")
+        self._is_enum = False
+        if "enum" in parameters_metadata:
+            self._enum = parameters_metadata.get("enum")
+            self._is_enum = True
+        if "cluster_elements" in parameters_metadata:
+            self._cluster_elements = []
+            for cluster_element in parameters_metadata.get("cluster_elements"):
+                self._cluster_elements.append(ClusterElements(cluster_element))
```

Comment:
`cluster_element` isn't in https://github.com/ni/grpc-device-scrapigen/pull/143 so I don't know if anything related to it is correct.